spec/features/search/access_search_page_spec.rb

```diff
@@ -14,4 +14,13 @@
     visit search_path
     expect(page).to have_field("search")
   end
+  
+  it "renders a list when @query is not empty" do
+    FactoryBot.create :person
+    FactoryBot.create(:person, first_name: 'Micha', last_name: 'Perscheid', title: 'Not doctor')
+    visit "#{search_path}?query=Mich&commit=Search"
+    expect(page).to have_link 'Dr. Michael Perscheid'
```

Comment:
i would say that technically the text (rather than the link) would suffice on the page to fulfill the test criterium
-> maybe change it to "have_text?"